spec/rails_helper.rb

```diff
@@ -6,6 +6,9 @@
 # Add additional requires below this line. Rails is not loaded until this point!
 require 'capybara/rails'
 
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.